experiment/test_config.yaml

```diff
@@ -763,6 +763,78 @@ jobs:
     args:
     - --gcp-project=ubuntu-image-validation
     sigOwners: ['sig-node']
+  # release branch jobs (stable2 and stable3)
+  ci-kubernetes-e2e-gce-gci-k8sstable3-alphafeatures:
+    interval: 6h
+    args:
+    - --env=KUBE_FEATURE_GATES=AllAlpha=true
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-alphafeatures:
+    interval: 2h
+    args:
+    - --env=KUBE_FEATURE_GATES=AllAlpha=true
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable3-ingress:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-ingress:
+    interval: 2h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable3-reboot:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-reboot:
+    interval: 2h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable3-default:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-default:
```

Comment:
`gci` is the old name of `cos` so we want to get rid of it.

I feel like we should use `cosstable1` here because these tests are for testing k8s so we want to use a stable cos image.

We can reuse the tests defined around https://github.com/kubernetes/test-infra/blob/192bec2574504c113464d380ab4b8cf7d642ee64/experiment/test_config.yaml#L251

Reply:
I want to use the default cos image behavior for release jobs though..